tests/unit/test_define_xml_reader.py

```diff
@@ -11,6 +11,10 @@
 )
 from cdisc_rules_engine.services.define_xml_reader import DefineXMLReader
 
+test_define_2_0_file_path: str = (
```

Comment:
Is it easy to have a test for version 2.0 and 2.1. If not I wouldn't worry about making changes in this ticket